### PR TITLE
docs(library): exercise detail/picker Magazine rework — spec, plan, mockups

### PR DESCRIPTION
**Pull request by iwash-alt**
## Summary
- Spec + plan + 3 locked mockups for the Phase-2 #3 rework (exercise detail / "how to" + exercise picker)
- Closes launch-readiness #4 (every selectable exercise shows its media, rendered correctly — no half-cut GIF)
- Five-phase delivery: this PR (docs), then PR B (filter chrome + pure hook), PR C (BleedFigure + ExerciseRow + LibraryThumb), PR D (pages), PR E (orchestrator + wire-in + delete legacy)

## Test plan
- [ ] Review spec — composition, decisions locked, ship plan
- [ ] Open the three mockup HTMLs in a browser; confirm the locked direction in each (B Bleed, P1 Editorial, edge-cases all locked)
- [ ] No code changes; CI is docs-only

🤖 Generated with [claude-flow](https://github.com/ruvnet/claude-flow)

### CODE REVIEW

File: docs/superpowers/mockups/2026-05-16-exercise-detail-directions.html [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+  <div class="page-head">
+    <div class="label">brainstorming · phase 2 #3 · detail layout</div>
+    <h1>Exercise detail — three directions</h1>
+    <p>Each phone shows the same exercise (Barbell Bench Press) in a different layout. Real Magazine tokens, real GIF, real form-cue text from <code>src/data/exercises.ts</code>. Click any phone to register a pick — but the conversation in the terminal is what carries the answer.</p>
+  </div>
+
+  <div class="row">
+
+    <!-- ───────────── A: Plate ───────────── -->
+    <div class="phone a" data-choice="a" onclick="toggleSelect(this)">
```
[the remark this answers is not in the document]
Comment (iwash-alt) [reply docs/superpowers/mockups/2026-05-16-exercise-detail-directions.html]: Fixed in ef888eb — dropped the unused `onclick="toggleSelect(this)"` attributes (and the dead `.selected` CSS in detail-directions.html). Mockups are now visual-only, matching the existing summary mockup convention (no JS, pick verbally).
